docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -4,115 +4,158 @@ title: Cellular
+To enable **Cat-M1** (LTE-M or eMTC) or **NB-IoT** network modes, a specialized **M2M** (Machine-to-Machine) SIM card is required, distinct from the standard SIM cards used in cellphones. However, for **GSM/GPRS 2G** connections, a standard SIM card can generally be used.
 
-#### Attaching an antenna
+# Enabling Cellular on your Meadow Application
 
-Finally, connect an LTE antenna (Rubber ducky or Dome) with the `X1` click board IPX connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert an **M2M** SIM card into the cell module.
+Using Cellular on Meadow, you will need to do three things:
 
-![Quectel BG95-M3/NimbeLink Skywire click board setup](images/bg95.jpeg){: .center-image :standalone}
+## Adding a cell.config.yaml file
 
-<p align="center" style="font-size: smaller;">Quectel BG95-M3 click board with Meadow F7v2 Feather setup</p>
+Create a **cell.config.yaml** file, set the `Copy To Output` property to `Copy always` or `Copy if newer`, and fill out the values under the `Settings` section like APN, Module, and a additional optional pins to ensure a proper connection with your internet provider. Here's what a cell config file looks like with the required and optional fields:
 
-### Quectel M95 - GSM2 click board
+```yaml
+Settings:
+    APN: YOUR-APN         # (required) Access Point Name
+    Module: BG95M3        # (required) Module model (BG770A, BG95M3 or M95)
+    User: USER            # (optional) APN user 
+    Password: PASSWORD    # (optional) APN password
+    Operator: 00000       # (optional) Carrier numeric operator code
+    Mode: CATM1           # (optional) Network mode (CATM1, NBIOT or GSM)
+    Interface: /dev/ttyS1 # (optional) Serial interface (UART1 (COM1) = /dev/ttyS0, UART4 (COM4) = /dev/ttyS1, UART6 = /dev/ttyS3) 
+    TurnOnPin: D10        # (optional) Enable pin to turn the module on/off. Default value is Meadow Pin name D10
+```
 
-To configure the hardware, start by connecting the necessary jumpers for communication between the Meadow device and the cell module. Then make the necessary connections to supply and turn on the cell module. Finally, connect an antenna to the click board.
+A few things to consider:
+ * If the carrier numeric operator code (**Operator**) or the network mode is not specified (**Mode**), the module will attempt to automatically determine the optimal network based on the M2M sim card inserted and your location. 
+ * **However, if you encounter any connectivity issues, we recomment to set the operator code value to the `Operator` property**. If you don't know your operator code, you can use the **Cell Network Scanner** method that will list nearby networks in the area.
+ * `TurnOnPin` is a pin used to turn on the module.
 
-#### Connecting serial pins (UART)
+## Specify Network Interface and reserved pins
```

Comment:
I liked the two cell.config.yamls samples, are very clear!

Reply:
:)